_overviews/parallel-collections/overview.md

```diff
@@ -206,7 +206,7 @@ variable.
 
 In the above example, it's possible for two threads to read the _same_ value
 in `sum`, to spend some time doing some operation on that value of `sum`, and
-then to attempt to write a new value to `sum`, potentially resulting in an
+then to attempt to write a new value to `sum`, potentially resulting in
```

Comment:
this change is incorrect. the original maybe awkward, but this change doesn't help